cli/azd/extensions/microsoft.azd.extensions/internal/resources/languages/javascript/build.ps1

```
@@ -61,6 +61,7 @@ foreach ($PLATFORM in $PLATFORMS) {
     }
 
     Write-Host "Installing dependencies..."
+    npm install -g pkg
```

Probably we should not install global packages on user's system w/o asking for confirmation.

Can we use npx instead? Or should we fail fast and ask user to install pkg first?  What happen, for example, if Node.js is not installed?

---

@vhvb1989 Thank you for the suggestion! I've updated the script to use `npx` to install pkg instead of installing it globally. The script also now checks whether `Node.js` and `npm` are installed, and it will exit with a message if they are missing.